app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue

```diff
@@ -178,12 +178,14 @@ export default {
         {
           field: 'company',
           key: 'company',
+          sortBy: this.sortConfig.company_name || '',
```

Comment:
Instead of `''` we could use `undefined`

Reply:
@nithindavid I have used it as we used it for other filters, e.g email, and phone_number. Should I change from `''` to `undefined` for the previous filters too?